src/engine/source/cmds/src/test.cpp

```diff
@@ -40,6 +42,7 @@ namespace cmd::test
 {
 void run(const Options& options)
 {
+    Metrics::instance().initMetrics("engine-metrics", "/var/ossec/engine/store/metrics/config/0");
```

Comment:
severe - desing: This is very dangerous. I think it shouldn't be like that. what do you think @cborla

Reply:
where we could initialize the metrics? Because we depend on the HFs of the KVDB, when wanting to use the test, it is required to initialize the metrics module.